doc_test.go

```diff
@@ -0,0 +1,35 @@
+package pusher_test
+
+import (
+	"context"
+
+	"github.com/pusher/pusher-platform-go/client"
+	"github.com/pusher/pusher-platform-go/instance"
+)
+
+func Example() {
+	// Instantiate
+	app, err := instance.New(instance.Options{
```

Comment:
We should prefer `instance` over `app`

Reply:
Wanted to avoid collision with package name :(